packages/arp/arp.0.1.1/opam

```diff
@@ -29,6 +29,11 @@ depopts: [
   "tcpip"
   "lwt"
 ]
+
+conflicts: [
+  "tcpip" {< "3.0.0"}
+]
```

Comment:
oh, ic - thanks for investigating (I remember I wrote arp with the then unreleased move of tcpip into smaller libraries in mind).  my intuition of `tcpip >= 3` and `mirage-types-lwt <3` is that this should be empty (post-mirage3 and pre-mirage3 things)... maybe best to remove `arp < 0.2.0`!?